hledger-budget does not compile

Using Haskell 8.2.1 on `master` branch (`d257351`) running `bin/compile.sh` results in this error:

```
building dependencies
building add-on commands
[1 of 1] Compiling Main             ( hledger-budget.hs, hledger-budget.o )

hledger-budget.hs:193:42: error:
    • Couldn't match expected type ‘InputOpts’
                  with actual type ‘CliOpts’
    • In the first argument of ‘ignore_assertions_’, namely ‘opts’
      In the second argument of ‘($)’, namely ‘ignore_assertions_ opts’
      In the expression: not $ ignore_assertions_ opts
    |
193 |     let assrt = not $ ignore_assertions_ opts
    |                                          ^^^^
```

Any help with this is appreciated.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/49933804-hledger-budget-does-not-compile?utm_campaign=plugin&utm_content=tracker%2F536505&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F536505&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/bin/hledger-budget.hs
+++ b/bin/hledger-budget.hs
@@ -190,7 +190,7 @@ cmdmode = (mainmode [])
 
 journalBalanceTransactions' :: CliOpts -> Journal -> IO Journal
 journalBalanceTransactions' opts j = do
-    let assrt = not $ ignore_assertions_ opts
+    let assrt = not . ignore_assertions_ $ inputopts_ opts
     either error' return $ journalBalanceTransactions assrt j
 
 withJournalDo' :: CliOpts -> (CliOpts -> Journal -> IO ()) -> IO ()

